_articles/Pop!_OS-20.04-LTS-Release-Notes.md

```diff
@@ -57,6 +57,7 @@ Pop!\_Shop now loads up to 20 Pop!\_Picks making highlighted applications more a
 Dark mode is enabled by default. Default desktop wallpaper has a new design. Both defaults can be changed within the Appearance settings.
 
 ### GNOME 3.36
+<<<<<<< HEAD:_articles/Pop!_OS-20.04-LTS-Release-Notes.md
```

Comment:
Why are there merge conflicts committed?

Reply:
I did not see/wasn't alerted to the conflict. I didn't make any edits to the release notes article.

Reply:
Or, I didn't see them until now I should say.

Reply:
It's fixed now, thanks